src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -1308,6 +1311,59 @@ public void setPackage(String package_) {
     String pdf;
   }
 
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class NextActionKonbiniDisplayDetails extends StripeObject {
+    /** The timestamp at which the pending Konbini payment expires. */
+    @SerializedName("expires_at")
+    Long expiresAt;
+
+    /**
+     * The URL for the Konbini payment instructions page, which allows customers to view and print a
+     * Konbini voucher.
+     */
+    @SerializedName("hosted_voucher_url")
+    String hostedVoucherUrl;
+
+    @SerializedName("stores")
+    NextActionKonbiniDisplayDetailsStores stores;
+  }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class NextActionKonbiniDisplayDetailsStores extends StripeObject {
```

Comment:
This is almost certainly not what we want. Should be just `public static class Stores` and be an inner class of `NextActionKonbiniDisplayDetails`